src/plugins/raven/actions/recievedCollaborationNodes.js

```diff
@@ -0,0 +1,6 @@
+import { actionTypes } from '../constants';
+
+export default (nodes) => ({
```

Comment:
@nanetteCodes lets call this `receiveCollaborationNodes`

Reply:
renamed it